django/publicmapping/redistricting/tasks.py

```diff
@@ -20,19 +20,19 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 
-Author: 
+Author:
     Andrew Jennings, David Zwarg
 """
 
-from celery.task import task
-from celery.task.http import HttpDispatchTask
+from publicmapping.celery import app
+# from celery.task.http import HttpDispatchTask
```

Comment:
If there's no or a low chance that we'll re-enable this later, I'd just delete it.